Move more reporters under crier (#15983)
kubernetes/test-infra: prow/cmd/crier/README.md

Similar to the [gerrit adapter](/prow/cmd/gerrit), you'll need to specify `--gerrit-projects` for
your gerrit projects, and also `--cookiefile` for the gerrit auth token (leave it unset for anonymous).

Gerrit reporter will send an aggregated summary message, when all [gerrit adapter](/prow/cmd/gerrit)

> scheduled prowjob finishes on a revision, aka, on `SuccessState`, `FailureState`, `AbortedState` or `ErrorState`.

Reviewer: ```suggestion
scheduled prowjobs with the same report label finish on a revision.
```

Reviewer: ^I think it is important to clarify how the grouping mechanism works.

It will also attach a report url so people can find logs of the job.

The reporter will also cast a +1/-1 vote on the `prow.k8s.io/gerrit-report-label` label of your prowjob,

> or by default it will vote on `CodeReview` label. Where `+1` means all jobs on the patshset pass and `-1`

Reviewer: Side note: I wish we defaulted to `Verfied` instead, but it is probably too late to change. It rarely makes sense for presubmit success to allow bypassing code review.
